dom.go

```diff
@@ -1612,6 +1661,14 @@ type BasicElement struct {
 	*BasicNode
 }
 
+func (e *BasicElement) Attributes() []Attribute {
+	return attributeListToAttributes(e.Get("attributes"))
+}
+
+func (e *BasicElement) Dataset() []Attribute {
+	return dataListToAttributes(e.Get("attributes"))
```

Comment:
Why does this use `attributes` and duplicated logic regarding the trimming of the data prefix, and not `dataset` directly?